src/fastmcp/server/tasks/protocol.py

```
@@ -0,0 +1,346 @@
+        async with docket.redis() as redis:
+            task_key_bytes = await redis.get(redis_key)
+            created_at_bytes = await redis.get(created_at_key)
+
+        task_key = None if task_key_bytes is None else task_key_bytes.decode("utf-8")
+        created_at = (
+            None if created_at_bytes is None else created_at_bytes.decode("utf-8")
+        )
+
+        if task_key is None:
+            # Task not found - raise error per MCP protocol
+            raise ValueError(f"Task {client_task_id} not found")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)